_data/team.yml

```
@@ -146,7 +146,7 @@ people:
   surname: Liu
   thumbnailUrl: Carpentries.png
   social: 
-      # - {name: "twitter", link: "#"}
+    - {name: "twitter", link: https://twitter.com/frank0434"}
```

fix missing quotation
```suggestion
    - {name: "twitter", link: "https://twitter.com/frank0434"}
```